assets/xml/scenes/overworld/spot18.xml

```diff
@@ -1,9 +1,17 @@
 <Root>
     <File Name="spot18_scene" Segment="2">
-        <Cutscene Name="gGoronCityDaruniaCorrectCs" Offset="0x59E0"/>
-        <Cutscene Name="gGoronCityDarunia01Cs" Offset="0x6930"/>
-        <Cutscene Name="gGoronCityDaruniaWrongCs" Offset="0x7DE0"/>
+        <Cutscene Name="gGoronCityDaruniaCorrectSongCs" Offset="0x59E0"/>
+        <Cutscene Name="gGoronCityDaruniaWrongSongCs" Offset="0x7DE0"/>
+
+        <!--
+            This cutscene's implementation makes this cutscene override any other cutscene if Darunia is present in the scene.
+            This was probably used as a debug tool while creating the final cutscene.
+        -->
```

Comment:
It doesn't seem appropriate to put this kind of comment here. It is not the cutscene itself that overrides other cutscenes, it is Darunia who is doing the overriding, so if the comment should exist it should live there instead.

Even then, the line `This was probably used as a debug tool while creating the final cutscene.` is too much speculation.

Reply:
yeah also agreed on not commenting on it too deeply. 
when i said to comment on it on discord, i just meant like "this cutscene isnt used in normal gameplay"

Reply:
I changed it again to say what you just said (I'm not very inspired lol), let me know if that should be updated again